.circleci/config.yml

```diff
@@ -357,3 +369,14 @@ workflows:
             - test_fmri_spm_nested_fsl_feeds
             - test_py3_fmri_fsl_spm
             - test_py3_fmri_spm_dartel_multiproc
+      - deploy_pypi:
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /.*/
```

Comment:
for tags - should we only do master?

Reply:
What's the goal you're trying to achieve? I assume not to push a package every time master is updated.

Reply:
i think i may be misunderstanding how circle bit works - i thought the tags section should look only for tags applied to the master branch, since tags could potentially be applied on any branch. i'll read up on my train ride in a bit.

Reply:
So by default Circle will build on all branches and no tags. In order to restrict it from some branches, you can match with `ignore` and `only`, so here I'm saying don't run this on any branches. And then to run on all tags (regardless of branch) we use `only: /.*/`.

Which is to say, this job will be run iff we have a tag.